dbtmetabase/__init__.py

```diff
@@ -1,785 +1,12 @@
-    )
-    @click.option(
-        "--dbt_includes",
-        metavar="MODELS",
-        cls=MultiArg,
-        type=list,
-        help="Model names to limit processing to",
-    )
-    @click.option(
-        "--dbt_excludes",
-        metavar="MODELS",
-        cls=MultiArg,
-        type=list,
-        help="Model names to exclude",
-    )
-    @click.option(
-        "--metabase_database",
-        envvar="MB_DATABASE",
-        show_envvar=True,
-        required=True,
-        cls=OptionAcceptableFromConfig,
-        type=click.STRING,
-        help="Target database name as set in Metabase (typically aliased)",
-    )
-    @click.option(
-        "--metabase_host",
-        metavar="HOST",
-        envvar="MB_HOST",
-        show_envvar=True,
-        required=True,
-        cls=OptionAcceptableFromConfig,
-        type=click.STRING,
-        help="Metabase hostname",
-    )
-    @click.option(
-        "--metabase_user",
-        metavar="USER",
-        envvar="MB_USER",
-        show_envvar=True,
-        cls=OptionAcceptableFromConfig,
-        type=click.STRING,
-        help="Metabase username",
-    )
-    @click.option(
-        "--metabase_password",
-        metavar="PASS",
-        envvar="MB_PASSWORD",
-        show_envvar=True,
-        cls=OptionAcceptableFromConfig,
-        type=click.STRING,
-        help="Metabase password",
-    )
-    @click.option(
-        "--metabase_session_id",
-        metavar="TOKEN",
-        envvar="MB_SESSION_ID",
-        show_envvar=True,
-        default=None,
-        cls=OptionAcceptableFromConfig,
-        type=click.STRING,
-        help="Metabase session ID",
-    )
-    @click.option(
-        "--metabase_http/--metabase_https",
-        "metabase_use_http",
-        default=False,
-        cls=OptionAcceptableFromConfig,
-        help="use HTTP or HTTPS to connect to Metabase. Default HTTPS",
-    )
-    @click.option(
-        "--metabase_verify",
-        metavar="CERT",
-        type=click.Path(exists=True, file_okay=True, dir_okay=False),
-        help="Path to certificate bundle used by Metabase client",
-    )
-    @click.option(
-        "--metabase_sync/--metabase_sync_skip",
-        "metabase_sync",
-        cls=OptionAcceptableFromConfig,
-        default=True,
-        help="Attempt to synchronize Metabase schema with local models. Default sync",
-    )
-    @click.option(
-        "--metabase_sync_timeout",
-        metavar="SECS",
-        type=int,
-        help="Synchronization timeout (in secs). If set, we will fail hard on synchronization failure; if not set, we will proceed after attempting sync regardless of success. Only valid if sync is enabled",
-    )
-    @click.option(
-        "--http_extra_headers",
-        cls=OptionAcceptableFromConfig,
-        type=(str, str),
-        multiple=True,
-        help="Additional HTTP request header to be sent to Metabase.",
-    )
```

Comment:
Hi, I see that this part has been removed from the new version 1.0.0, is it expected?

Reply:
Still available in the programmatic API. I couldn't get the dictionary type to work via CLI flag, were you using it via config YAML file?

Reply:
Released in v1.0.1, you can use `--http-header x-some-header some-value` or via YAML config:

```yaml
config:
  http_headers:
    - [x-some-header, some-value]
```